pkg/chartutil/create_test.go

```diff
@@ -57,6 +57,7 @@ func TestCreate(t *testing.T) {
 		NotesName,
 		ServiceName,
 		TemplatesDir,
+		TemplatesTestsDir,
```

Comment:
Can you also add: ServiceAccountName & TestConnectionName ?

Reply:
done